pkg/api/pod/util.go

```diff
@@ -88,14 +88,19 @@ type Visitor func(name string) (shouldContinue bool)
 // VisitPodSecretNames invokes the visitor function with the name of every secret
 // referenced by the pod spec. If visitor returns false, visiting is short-circuited.
 // Transitive references (e.g. pod -> pvc -> pv -> secret) are not visited.
+// allContainers, when true, specifies that all containers be visited.
```

Comment:
as a caller, this is hard to understand... I would either make the arg match VisitContainers and just pass it through, or explain the parameter in terms of the arg passed to VisitContainers and refer to the docs for that function

Reply:
I made the parameter a pass-through